libs/test-utils/src/fake_kiosk.ts

```diff
@@ -99,6 +99,8 @@ export function fakeKiosk({
     sign: jest.fn(),
     log: jest.fn(),
     reboot: jest.fn(),
+    // eslint-disable-next-line vx/gts-identifiers
+    rebootToBIOS: jest.fn(),
```

Comment:
Nit: I know that it's weird to follow the camel case convention for acronyms, but the rest of the code does it (e.g. see `Usb` below and `Ipp` elsewhere), so I don't know that there's a strong reason to make an exception here.

Reply:
grumble... you are right, even as I find this particular decision annoying :)